codelabs/codelab-02/README.md

```diff
@@ -494,7 +494,7 @@ Submit this assignment to `codelab2` on the submit server. Upload a zipped direc
 ```
 <directory id>.zip
 upload.py
```

Comment:
indent

Reply:
for future ref, you can just cross-check the codelabs from last semester with https://umd-cs-stics.gitbooks.io/cmsc389l-fall2017/content/lectures/lecture-02/codelab.html